conf/messages

```diff
@@ -257,12 +257,16 @@ linkClients.error.no-radio-selected= Select yes if you want to link your clients
 #Subscription complete
 subscriptionComplete.title=Agent services account created
 subscriptionComplete.h1=Agent services account created
-subscriptionComplete.accountName=Your account reference number
+subscriptionComplete.accountName=You created an account for
 subscriptionComplete.h2=What you can do next
+subscriptionComplete.h3=Help us improve this service
 subscriptionComplete.p1=Make a note of your account reference number <strong class="bold">{0}</strong>. You will need this if you are not able to access your agent services account.
-subscriptionComplete.p2=<a href="{0}">Check the guidance</a> to find out how to:
-subscriptionComplete.bullet-list.1=sign in to your agent services account
-subscriptionComplete.bullet-list.2=use new HMRC services on behalf of your clients
+subscriptionComplete.p2=You can:
+subscriptionComplete.p3=What did you think of this service?
+subscriptionComplete.p4= (takes 30 seconds)
```

Comment:
is space ok after = ?

Reply:
Yes we need the space to render the view correctly.